datadog_checks_dev/datadog_checks/dev/tooling/specs/configuration/consumers/model.py

```diff
@@ -34,7 +34,13 @@ def example_looks_informative(example):
 
 
 def get_default_value(type_data):
-    if 'default' in type_data:
+    if 'display_default' in type_data:
+        display_default = type_data['display_default']
+        if display_default is None:
+            return NO_DEFAULT
+        else:
+            return display_default
+    elif 'default' in type_data:
         return type_data['default']
```

Comment:
I was thinking and we should flip these since, as in the PR you linked, `default` should be the ultimate source of truth for runtime values:

```suggestion
    if 'default' in type_data:
        return type_data['default']
    elif 'display_default' in type_data:
        display_default = type_data['display_default']
        if display_default is None:
            return NO_DEFAULT
        else:
            return display_default
```